`--vispy-audit` broken after adding freezing into Vispy

the `ImageTester` tries to attach a `self.grid` after calling `super`'s init. 


Error log on using `--vispy-audit`:

```
../../anaconda/lib/python3.4/site-packages/numpy/testing/decorators.py:146: in skipper_func
    return f(*args, **kwargs)
vispy/scene/cameras/tests/test_perspective.py:49: in test_perspective_render
    max_px_diff=200)
vispy/testing/image_tester.py:140: in assert_image_approved
    _get_tester().test(image, std_image, message)
vispy/testing/image_tester.py:65: in _get_tester
    tester = ImageTester()
vispy/testing/image_tester.py:296: in __init__
    self.grid = self.central_widget.add_grid()
vispy/util/frozen.py:16: in __setattr__
    'attributes' % (key, self))
E   AttributeError: 'grid' is not an attribute of class <ImageTester (PyQt4) at 0x1091780f0>. Call "unfreeze()" to allow addition of new attributes
========================================================== 1 failed in 1.19 seconds ==========================================================
```

So, using `--vispy-audit` causes Vispy to fail inside `ImageTester`

--- a/vispy/testing/image_tester.py
+++ b/vispy/testing/image_tester.py
@@ -291,7 +291,13 @@ class ImageTester(scene.SceneCanvas):
     """Graphical interface for auditing image comparison tests.
     """
     def __init__(self):
+        self.grid = None
+        self.views = None
+        self.console = None
+        self.last_key = None
+
         scene.SceneCanvas.__init__(self, size=(1000, 800))
+
         self.bgcolor = (0.1, 0.1, 0.1, 1)
         self.grid = self.central_widget.add_grid()
         border = (0.3, 0.3, 0.3, 1)
@@ -303,9 +309,14 @@ class ImageTester(scene.SceneCanvas):
             v.camera = 'panzoom'
             v.camera.aspect = 1
             v.camera.flip = (False, True)
+            # unfreeze it to set the image and label on the view
+            # this is slightly hacky, but it is simpler than
+            # creating another class/storing as a dict or a tuple
+            v.unfreeze()
             v.image = scene.Image(parent=v.scene)
             v.label = scene.Text(label_text[i], parent=v, color='yellow',
                                  anchor_x='left', anchor_y='top')
+            v.freeze()
 
         self.views[1].camera.link(self.views[0].camera)
         self.views[2].camera.link(self.views[0].camera)
@@ -331,7 +342,6 @@ class ImageTester(scene.SceneCanvas):
         self.views[2].image.set_data(diff)
         self.views[0].camera.set_range()
 
-        self.last_key = None
         while True:
             self.app.process_events()
             if self.last_key is None:

